bindgen-citro3d/src/main.rs

```diff
@@ -0,0 +1,80 @@
+use std::iter::FromIterator;
+use std::path::PathBuf;
+
+use bindgen::callbacks::{DeriveTrait, ImplementsTrait, ParseCallbacks};
+use bindgen::{Builder, RustTarget};
+
+fn main() {
+    let devkitpro = std::env::var("DEVKITPRO").expect("DEVKITPRO not set in environment");
+    let devkitarm = std::env::var("DEVKITARM").expect("DEVKITARM not set in environment");
+
+    let include_path = PathBuf::from(devkitpro).join(PathBuf::from_iter(["libctru", "include"]));
```

Comment:
nit: could this be: `PathBuf::from_iter([devkitpro.as_str(), "libctru", "include"])`